components/AvalancheCenterLogo.tsx

```diff
@@ -96,3 +97,45 @@ export const AvalancheCenterLogo: React.FunctionComponent<AvalancheCenterLogoPro
   actualStyle.aspectRatio = sizes[avalancheCenterId].width / sizes[avalancheCenterId].height;
   return images[avalancheCenterId](actualStyle);
 };
+
+export const preloadAvalancheCenterLogo = async (avalancheCenter: AvalancheCenterID) => {
```

Comment:
what does prefetching buy you here? these are local assets, not network ones

Reply:
(I guess you had already answered that question...surprised that it's slow to load, I wonder if that's a debug artifact)

Reply:
Yeah ... no idea how/why this was needed but it lagged/stuttered before.